timescaledb/how-to-guides/continuous-aggregates/migrate.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Migrate a continuous aggregate to the new form
+excerpt: Migrate old continuous aggregates to the new form introduced in TimescaleDB 2.7
+keywords: [continuous aggregates]
+---
+
+# Migrate a continuous aggregate to the new form
+
+In TimescaleDB 2.7 and above, continuous aggregates use a new format that
+improves performance and makes them compatible with more SQL queries. Continuous
+aggregates created in older versions of TimescaleDB, or created in a new version
+with the option `timescaledb.finalized` set to `false`, use the old format.
+
+To migrate a continuous aggregate from the old format to the new format, you can
+use the provided procedure. Connect to your database and run:
```

Comment:
```suggestion
use this command. Connect to your database and run:
```

It's not really a procedure?

Reply:
Yes, it is a procedure. Maybe we can write `use the provided Postgres procedure.` WDYT?

Reply:
Ah, I see what you mean. A procedure in the Postgres sense, not the documentation sense 😅 

```suggestion
use this procedure. Connect to your database and run:
```